scripts/optimize.sh

```diff
@@ -1,4 +1,4 @@
-docker run --rm -v "$(pwd)":/code \
-    --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
-    --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-    cosmwasm/workspace-optimizer-arm64:0.12.11
\ No newline at end of file
+docker run --rm -v "$(pwd)":/code --platform linux/amd64 \
+	--mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
+	--mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
+	cosmwasm/workspace-optimizer:0.12.11
```

Comment:
perhaps this script can be improved later to detect ARM  hosts (m1/m2) and run with different docker image instead.